src/client/entity-editor/submission-section/actions.js

```diff
@@ -81,8 +80,7 @@ function postSubmission(url: string, data: Map<string, mixed>): Promise {
 	 */
 
 	const [, submissionEntity] = url.split('/');
-	return request.post(url).send(data)
-		.promise()
+	return request.post(url).send(Object.fromEntries(data))
```

Comment:
Either `superagent` or `superagent-bluebird-promise` seems to have coerced `Map`s to plain JS objects previously, but this isn't the case with the update, so we do it ourselves.